app/tools/android/make_apk_test.py

```diff
@@ -370,7 +389,7 @@ def testAppBigVersionCodeBase(self):
            '--description=a sample application',
            '--app-versionCodeBase=3000000',
            arch,
-           '--app-url=http://www.intel.com',
+           '--app-url=http://www.crosswalk-project.org',
```

Comment:
I think Halton meant you should do this for the new code you are adding. Changing all the existing occurrences is unrelated to what your patch intends to do, and only makes it bigger; I support changing these existing URLs, but separately.